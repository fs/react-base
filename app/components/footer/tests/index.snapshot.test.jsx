import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../';

describe('Footer', () => {
  const renderComponent = () => renderer.create(<Footer />);

  it('renders correctly', () => {
    expect(renderComponent().toJSON()).toMatchSnapshot();
  });
});
