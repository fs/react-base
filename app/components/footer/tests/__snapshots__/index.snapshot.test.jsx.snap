// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="footer"
>
  <p
    className="footer-text"
  >
    React-base
  </p>
</footer>
`;
